injector/src/main/java/edu/ucr/cs/riple/injector/Helper.java

```diff
@@ -212,15 +213,28 @@ private static boolean isDeclarationWithName(Node node, String name) {
   }
 
   /**
-   * Locates an anonymous class at specific index.
+   * Locates an anonymous class or enum constant at specific index.
    *
    * @param cursor Starting node for traversal.
    * @param index index.
-   * @return anonymous class at specific index.
+   * @return anonymous class or enum constant at specific index.
    * @throws TargetClassNotFound if the target class is not found.
    */
-  private static Node findAnonymousClass(Node cursor, int index) throws TargetClassNotFound {
+  private static Node findAnonymousClassOrEnumConstant(Node cursor, int index)
+      throws TargetClassNotFound {
     final List<Node> candidates = new ArrayList<>();
+    if (cursor instanceof EnumDeclaration) {
+      // According to the Java language specification, enum constants are the first members of
+      // enums. In
```

Comment:
Might need some manual adjusting here to properly reformat this comment.

Reply:
Fixed in https://github.com/ucr-riple/NullAwayAnnotator/pull/125/commits/ed14b51e7fd9f5d0080de1ba9167f6356e9d8b47